### PR TITLE
Fix search bar style in conference pages

### PR DESCRIPTION
**Pull request by OmeGak**
Merge alongside with https://github.com/indico/indico-plugins/pull/38.

Examples:

---

screenshot from 2016-11-28 15 51 10

---

screenshot from 2016-11-28 15 51 22

### CODE REVIEW

File: indico/htdocs/css/Conf_Basic.css [hunk 1]
```diff
@@ -302,40 +293,6 @@ pre {
     font-size: 11pt;
 }
 
-    /* Styles for the searchbox */
-
-.confSearchBox {
-    width: 250px;
-    float: right;
-    height: 24px;
-    background: transparent url("../images/conf/sprites_blue.png") no-repeat 0 0;
```
Comment (ThiefMaster) [inline indico/htdocs/css/Conf_Basic.css]: Is it used anywhere else?  Otherwise let's get rid of it

File: indico/htdocs/sass/base/_layout.scss [hunk 2]
```diff
@@ -136,7 +136,14 @@ body > * {
     }
 }
 
-/* Flexbox layout */
+
+// -- Flexbox layout -----------------------------------------------------------
```
Comment (ThiefMaster) [inline indico/htdocs/sass/base/_layout.scss]: Maybe the presence of such a comment is a good indicator that the block below belongs into a separate file?

Comment (OmeGak) [inline indico/htdocs/sass/base/_layout.scss]: Not a strong opinion. Thoughts? @pferreir @nop33

File: indico/htdocs/css/Conf_Basic.css (hunk 1 above)
Comment (OmeGak) [inline indico/htdocs/css/Conf_Basic.css]: It's still used in `.simpleTextAnnouncement`. :(

Comment (ThiefMaster) [inline indico/htdocs/css/Conf_Basic.css]: Maybe... just maybe... if it doesn't take too much time to clean up? ;)

File: indico/htdocs/sass/base/_layout.scss (hunk 2 above)
Comment (pferreir) [inline indico/htdocs/sass/base/_layout.scss]: No strong opinion.

Comment (ThiefMaster) [inline indico/htdocs/sass/base/_layout.scss]: It reminds me a log of the `################ ...` lines in legacy python code

Comment (nop33) [inline indico/htdocs/sass/base/_layout.scss]: I'd keep the simple `/* */` I agree it's pretty ugly having an asymmetric line of dashes

Comment (nop33) [inline indico/htdocs/sass/base/_layout.scss]: And yes, maybe a different file wouldn't be such a bad idea. OTOH it's the `layout.scss`, so the flexbox classes fit just fine there.

Comment (ThiefMaster) [inline indico/htdocs/sass/base/_layout.scss]: FWIW I'd use the SCSS-style `//` comments. It's just one line anyway.

**Pull request merged**